extensions/mssql/package.json

```diff
@@ -419,6 +419,29 @@
     "connectionProvider": {
       "providerId": "MSSQL",
       "displayName": "Microsoft SQL Server",
+      "iconPath": [
+        {
+          "condition": "serverInfo.isCloud",
+          "path": {
+            "light": "resources/light/azureDB.svg",
+            "dark": "resources/dark/azureDB_inverse.svg"
+          }
+        },
+        {
+          "condition": "serverInfo.options.isBigDataCluster",
+          "path": {
+            "light": "resources/light/sql_bigdata_cluster.svg",
+            "dark": "resources/dark/sql_bigdata_cluster_inverse.svg"
+          }
+        },
+        {
+          "condition": "true",
```

Comment:
I'd suggest we don't need this option. This should be the fallback if no other icon condition is true. If other conditions fail, we fall back to the default_server which ships inside the box. That avoids duplicating the icons, and is going to be consistent across all connection types.

Reply:
fixed.